ui/src/components/yaml/index.tsx

```diff
@@ -377,35 +380,45 @@
                   />
                 </Space>
               </div>
-              <div className={styles.yaml_content_diagnosisContent}>
-                {interpretStatus === 'loading' ||
-                (interpretStatus === 'streaming' && !interpret) ? (
-                  <div className={styles.yaml_content_diagnosisLoading}>
-                    <Spin />
-                    <p>{t('EventAggregator.DiagnosisInProgress')}</p>
-                  </div>
-                ) : interpretStatus === 'streaming' ? (
-                  <div>
-                    <Markdown>{interpret}</Markdown>
-                    <div className={styles.yaml_content_streamingIndicator}>
-                      <span className={styles.dot}></span>
-                      <span className={styles.dot}></span>
-                      <span className={styles.dot}></span>
+              <div className={styles.yaml_content_diagnosisBody}>
+                <div
```

Comment:
The error state now uses an Alert component with type 'error', which provides better visual feedback to users.